README.md

```diff
@@ -20,6 +20,62 @@ Note the address bar indicating google.com. Also, since the content of USA Today
 - Can't request permissions
 - Can't be added to homescreen
 
+## Goals and non-goals
+
+### Goals
+
+* **authenticated-sharing**: Alice can download a package containing
+  a [PWA](https://developers.google.com/web/progressive-web-apps/checklist) from
+  origin `O`, and transmit it to their peer Bob, and then Bob can install
+  the PWA with a proof that it came from `O`.
+* **streamed-loading**: When a package is downloaded over TLS from its original
```

Comment:
I think it can be simplified by dropping 'TLS' requirement since unsigned content also makes sense. The TLS can be used in lieu of signature verification (but even that is more complicated, with countersigning) - so lets just keep streamed-loading as a goal regardless trust issues.